include/otbObiaLSBaatzSegmentationScheduler.txx

```diff
@@ -551,49 +620,53 @@ LSBaatzSegmentationScheduler<TInputImage>
                         }
 
                         // Compute the offset of displacement.
```

Comment:
for sake of code clarity I would add something like ``uint64_t blockSize = sizeof(size_t) + m_MaxNumberOfBytes`` and reuse it L623,L626,L630,L634, ...